standard-page.css

```diff
@@ -11,13 +11,18 @@
 }
 
 .h5p-standard-page .h5p-accordion .h5p-panel-title {
-  padding: 0.8em 0 0.8em 1.25em;
+	padding: 0.2em 0 0.2em 1em;
```

Comment:
why does the indent changed ? It's 2 spaces on my IDE

Reply:
Perhaps it actually stores it as tab, but visualizes it as 2 spaces?

Reply:
Oh smart, that's possible I suppose 🤔

Reply:
but It doesn't in other files ...